test/viaq-data-model.sh

```diff
@@ -115,6 +115,7 @@ get_logmessage2() {
 }
 
 # TEST 1
+TIMEOUT=600
```

Comment:
it really shouldn't take this long - that would indicate that there is some other problem

Is it the first `wait_for_fluentd_to_catch_up` that fails without the timeout?  Is it a specific line in the test that is failing?

One thing I have noticed is that some of the tests do not clean up correctly - specifically, they do not wait until mux and fluentd are up and running in some cases.  So, for example, if you see that it is the first test that fails, maybe it is because fluentd is not yet up and running?

Reply:
> Is it the first wait_for_fluentd_to_catch_up that fails without the timeout? Is it a specific line in the test that is failing?

Sometimes, yes.  But more often, {"message":"GET /$UUID 404 "} fails.

> One thing I have noticed is that some of the tests do not clean up correctly

Ok.  Let me try just running test-viaq-data-model.sh a couple of times...  (I think I ran into it with the single test case, but not 100% sure...)